IPython/core/completer.py

```diff
@@ -1879,6 +1891,14 @@ def _complete(self, *, cursor_line, cursor_pos, line_buffer=None, text=None,
         self.line_buffer = line_buffer
         self.text_until_cursor = self.line_buffer[:cursor_pos]
 
+        # Do magic arg matches
+        for matcher in self.magic_arg_matchers:
+            matches = [(m, matcher.__qualname__) for m in matcher(line_buffer)]
```

Comment:
I don't think we need to put `matcher.__qualname__` into the list - it's going to be the same for every item anyway.